sceptre/synapsedev/config/prod/snowflake-access.yaml

```diff
@@ -1,9 +1,11 @@
 template:
   type: http
-  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.8.2/IAM/snowflake-bucket-access.yaml
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.8.9/IAM/snowflake-bucket-access.j2
 stack_name: snowflake-access
 parameters:
-  BucketArn: arn:aws:s3:::dev.datawarehouse.sagebase.org
-  BucketPrefix: warehouse
   SnowflakeAccountArn: arn:aws:iam::365909334157:user/m2nb0000-s
   SnowflakeAccountExternalId: !ssm /infra/SnowflakeAccountExternalId
+sceptre_user_data:
+  Buckets:
```

Comment:
Is this for an existing connection? Each s3 bucket needs separate policies.

Reply:
yes, existing connection.  just add another bucket to the `Buckets` list when setting up new connections.

Reply:
Thanks, I know there is also a production bucket somewhere, will that be updated in a separate PR?

Reply:
yes, add another bucket to the `Buckets` list in a follow on PR.